src/VirtoCommerce.ContentModule.Web/Controllers/Api/ContentController.cs

```diff
@@ -245,8 +246,16 @@ public async Task<ActionResult> Unpack(string contentType, string storeId, [From
         [Route("folder")]
         [Authorize(Permissions.Create)]
         [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
```

Comment:
May be 
`[ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)] `
?

Reply:
Nope, we return array of error objects.

Reply:
Ok